cardinal/events/events_test.go

```diff
@@ -28,7 +27,7 @@ type Event struct {
 func TestEvents(t *testing.T) {
 	// broadcast 5 messages to 5 clients means 25 messages received.
 	numberToTest := 5
-	tf := testutils.NewTestFixture(t, nil, cardinal.WithDisableSignatureVerification())
+	tf := testutils.NewTestFixtureWithPort(t, nil, "1337", cardinal.WithDisableSignatureVerification())
```

Comment:
I made a ticket for myself to improve how we do port selection: https://linear.app/arguslabs/issue/WORLD-960/update-test-fixture-code-to-make-port-selection-more-robust